test/bigtest/tests/title-show-package-filter-test.js

```diff
@@ -72,13 +72,84 @@ describe('TitleShow package filter flow', () => {
         expect(PackageFilterModal.modalIsDisplayed).to.be.true;
       });
 
+      it('multi select accordion should be opened by default', () => {
+        expect(PackageFilterModal.packageMultiSelectAccordion.isOpen).to.be.true;
+      });
+
+      it('should display the package filter multi select', () => {
+        expect(PackageFilterModal.packageMultiSelect.isPresent).to.be.true;
+      });
+
       it('the package filter select should be empty by default', () => {
         expect(PackageFilterModal.packageMultiSelect.values()).to.deep.equal([]);
       });
 
+      it('selection filter accordion should be opened by default', () => {
+        expect(PackageFilterModal.selectionFilterAccordion.isOpen).to.be.true;
+      });
+
+      it('should display the package filter selection filter', () => {
+        expect(PackageFilterModal.selectionFilterIsPresent).to.be.true;
+      });
+
+      describe('Selection filter', () => {
+        describe('Selection filter selection status all', () => {
```

Comment:
Better to call block of test with `when` word, for exp. 
```suggestion
        describe('when status "All" selected for selection filter', () => {
```

Reply:
Fixed